BasementRenovator.py

```diff
@@ -809,6 +820,47 @@ def paintEvent(self, event):
                 int(Qt.AlignRight | Qt.AlignBottom),
                 f"Base HP : {e.entity.config.baseHP}",
             )
+            textY += 16
+
+            if e.entity.config.stageHP is not None and e.entity.config.stageHP != "0":
+                painter.drawText(
+                    r.right() - 34 - 200,
+                    textY,
+                    200,
+                    12,
+                    int(Qt.AlignRight | Qt.AlignBottom),
+                    f"Stage HP : {e.entity.config.stageHP}",
+                )
+                textY += 16
+
+                stageHPNum = mainWindow.floorInfo.get(
+                    "StageHPNum"
+                ) or mainWindow.floorInfo.get("Stage")
```

Comment:
Mind adding a note about this to `NicheFeatures.md`? Especially since it's unused in the base xmls atm

Reply:
I added it to README.md since that is where the other variables for stage definitions are described. Made it clear it was an optional one though